Documentation/CRDs/ceph-client-crd.md

```diff
@@ -5,14 +5,18 @@ title: Client CRD
 Rook allows creation and updating clients through the custom resource definitions (CRDs).
 For more information about user management and capabilities see the [Ceph docs](https://docs.ceph.com/docs/master/rados/operations/user-management/).
 
-## Use Case
+## Use Case: Connecting to Ceph
 
 Use Client CRD in case you want to integrate Rook with applications that are using LibRBD directly.
 For example for OpenStack deployment with Ceph backend use Client CRD to create OpenStack services users.
 
 The Client CRD is not needed for Flex or CSI driver users. The drivers create the needed users automatically.
 
-## Creating Ceph User
+### Prerequisites
+
+This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](../Getting-Started/quickstart.md).
+
+### 1. Creating Ceph User
 
 To get you started, here is a simple example of a CRD to configure a Ceph client with capabilities.
```

Comment:
This example creates two CephClient CRs. We just need a single one for the example, right?

Reply:
These CephClient CRs are from the previous documentation -- I'm not sure what the intent for use for the previous ones was... It seems like a profile that works with OpenStack Cinder and Glance? I don't use OpenStack so I'm not sure if someone who does would have found that config useful.

Is it fine that I delete them and replace with the one for this specific case?

Reply:
OK so it doesn't really work to have `sqlitevfs` be the name for an example case -- so I'll switch it back to `example` everywhere, and replace the existing clients.